ulib/FStar.UInt128.fsti

```diff
@@ -62,7 +62,6 @@ val lognot: a:t -> Pure t
   (requires True)
   (ensures (fun r -> v r == lognot (v a)))
 
-#set-options "--lax"
```

Comment:
Removing this option in the interface fixes an unrelated bug (thanks to @tahina-pro, who figured out this was problematic!) that causes the implementation to get checked in lax mode, which is extremely confusing.